features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx

```diff
@@ -303,6 +307,18 @@ class Credentials extends React.Component {
                         </InlineMessage>
                     </>
                 );
+            } else if (isOnlyMutualSSL || isOnlyBasicAuth) {
```

Comment:
Don't we need to give exact reason why this is disabled?

Reply:
Message says 'Subscription is not required for Mutual SSL APIs or APIs with only Basic Authentication.'